website/static/js/fangorn.js

```diff
@@ -306,6 +306,7 @@ function _fangornCanDrop(treebeard, item) {
     if (canDrop === null) {
         canDrop = item.data.provider && item.kind === 'folder' && item.data.permissions.edit;
     }
+    canDrop = canDrop && window.contextVars.uploadInstruction;
```

Comment:
Using `contextVars` in a reusable module makes testing and reusability difficult. Can you use a different value, e.g. check whether uploads are enabled?